org.jacoco.core.test.validation.groovy/pom.xml

```diff
@@ -61,7 +59,11 @@
                     </execution>
                 </executions>
                 <configuration>
-                    <targetBytecode>${bytecode.version}</targetBytecode>
+                    <!--
+                    Force target bytecode to 1.7 for now
+                    as Groovy does not understand values such as 8 or 9
```

Comment:
Same for Kotlin, so we used Kotlin-specific property per bytecode profile - see https://github.com/jacoco/jacoco/blob/811cf32772ce5d36947235d11d3b5af6ee3f1d8b/org.jacoco.core.test.validation/pom.xml#L121